packages/swingset-liveslots/src/vatDataTypes.d.ts

```diff
@@ -73,6 +73,41 @@ export type DefineKindOptions<C> = {
   finish?: (context: C) => void;
 
   /**
+   * If provided, it describes the shape of all state records of instances
+   * of this kind.
+   */
+  stateShape?: StateShape;
+
+  /**
+   * If a `receiveAmplifier` function is provided, it will be called during
+   * definition of the exo class kit with an `Amplify` function. If called
+   * during the definition of a normal exo or exo class, it will throw, since
+   * only exo kits can be amplified.
```

Comment:
Here are some semantics that make the API clearer to me.  I hope this is what you intended.  If not, please consider this change to the implementation:
```suggestion
   * If a `receiveAmplifier` function is provided to an exo class kit definition,
   * it will be called with an `Amplify` function. If provided to the definition of
   * a normal exo or exo class, the definition will throw, since only exo kits
   * can be amplified.
```

Reply:
Yes. Done.